CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Removed deprecated API `RealmObject.removeChangeListeners()`. Use `RealmObject.removeAllChangeListeners()` instead.
 * `SyncUser.Callback` to becomes generic.
 * Removed `SyncUser.getAccessToken` method from public API, and rename it to `getRefreshToken`.
+* Removed `UNSUPPORTED_TABLE`, `UNSUPPORTED_MIXED` and `UNSUPPORTED_DATE` from `RealmFieldType`.
```

Comment:
Isn't this `internal` or something?

Reply:
No, this class moved into the public API with the migration API, but at that time we still had some internal types visible.  I doubt this will affect anyone, but public API = Changelog entry.

Reply:
Oh, okay!